src/workerd/api/node/crypto.c++

```diff
@@ -10,8 +10,13 @@
 #include <workerd/api/crypto/spkac.h>
 #include <workerd/jsg/jsg.h>
 
+#include <ncrypto.h>
+
 namespace workerd::api::node {
 
+// ======================================================================================
+#pragma region KDF
```

Comment:
What's the benefit of having pragma region?

Reply:
vscode makes these sections collapsible/foldable in the UI